project.json

```diff
@@ -9,6 +9,9 @@
         "config": ".verdaccio/config.yml",
         "storage": "tmp/local-registry/storage"
       }
+    },
+    "local-publish": {
+      "command": "npx nx run-many -t publish --registry=http://localhost:4873/ --ver={args.ver} --tag={args.tag}"
```

Comment:
This by itself is not enough. You also have to update `publish` target in package's `project.json` and update (use corelibrary version) `publish.mjs` in `tools/scripts`